tests/utils/utils.go

```diff
@@ -102,8 +102,13 @@ func CreateDefaultRelayerConfig(
 	fundedAddress common.Address,
 	relayerKey *ecdsa.PrivateKey,
 ) config.Config {
+	logLevel := logging.Info
+	if os.Getenv("LOG_LEVEL") == "debug" {
```

Comment:
```suggestion
	if os.Getenv("LOG_LEVEL") == logging.Debug.LowerString() {
```

Reply:
actually should we do default to `logging.Info`, if env variable is not empty string, set to `logging.ToLevel`?